PROJECT

```diff
@@ -0,0 +1,20 @@
+domain: kiegroup.org
```

Comment:
What is the purpose of this file?

Reply:
This the file created and used by the operator-sdk cli to create CSV and validate the bundle, it has meta information like our project name and the API we expose

Reply:
Ok, do we need to persist it on GitHub? If it is generated by operator-sdk then we can IMHO exclude it in gitignore.

Reply:
This file is not generated everytime, only once when we scaffold the project. And without it all CSV and buncle creation command will fail with `FATA[0000] error reading configuration: open PROJECT: no such file or directory`

Reply:
aha, ok, thanks for the context :)